### PR TITLE
fix: updated asset balances after sending transactions

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* fix: updated asset balances after sending transactions
  * feat: use vuex state to manage updating of balances
    * ref: https://github.com/AstarNetwork/astar-apps/pull/466#issuecomment-1190047497 https://github.com/AstarNetwork/astar-apps/pull/469/commits/83ffbae201262cfd1ebef56cb5e8bcb7ce4a3f62 @bobo-k2   
  * refactor: remove `useXcmAssets` since assets data is managed by vuex
 * refactor: removed duplicated logic in `computed properties`
   * feat: defined duplicated vuex logic in `useNetoworkInfo` hook

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

** fixes **
* fix: updated asset balances after sending transactions

=Before=
image

=After=
image

### CODE REVIEW

File: src/hooks/useChainInfo.ts [hunk 1]
```diff
@@ -53,6 +54,8 @@ export function useChainInfo(api: ApiPromise) {
     chainInfo.value = createInfo(api, systemChain, systemName, specName);
   });
 
+  const store = useStore();
```
Comment (sirius651) [inline src/hooks/useChainInfo.ts:57]: do we need store here?

File: src/hooks/useNetworkInfo.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { useStore } from 'src/store';
+import { computed } from 'vue';
+
+export function useNetworkInfo() {
+  const store = useStore();
+
+  const isMainnet = computed<boolean>(() => {
+    const chainInfo = store.getters['general/chainInfo'];
+    const network = chainInfo ? chainInfo.chain : '';
+    const isTestnet = network === 'Development' || network === 'Shibuya Testnet';
```
Comment (sirius651) [inline src/hooks/useNetworkInfo.ts:17]: maybe we should define this network name as external constant value

File: src/v2/services/IXcmService.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import { Asset } from 'src/v2/models';
+import { XcmAssets } from './../../store/assets/state';
```
Comment (sirius651) [inline src/v2/services/IXcmService.ts:1]: just can go through `src/store/assets/state` ?

File: src/hooks/useChainInfo.ts (hunk 1 above)
Comment (impelcrypto) [reply src/hooks/useChainInfo.ts]: Fixed in this commit https://github.com/AstarNetwork/astar-apps/pull/469/commits/ed79f2ae9bf5c316d52f3683f83024e8673eb5ca

File: src/hooks/useNetworkInfo.ts (hunk 2 above)
Comment (impelcrypto) [reply src/hooks/useNetworkInfo.ts]: I have fixed in this commit https://github.com/AstarNetwork/astar-apps/pull/469/commits/ed79f2ae9bf5c316d52f3683f83024e8673eb5ca

File: src/v2/services/IXcmService.ts (hunk 3 above)
Comment (impelcrypto) [reply src/v2/services/IXcmService.ts]: I have fixed in this commit https://github.com/AstarNetwork/astar-apps/pull/469/commits/ed79f2ae9bf5c316d52f3683f83024e8673eb5ca

**Review by sirius651 — APPROVED** [review]

**Review by hoonsubin — APPROVED** [review]

**Pull request merged**